windows/win_file.py

```diff
@@ -56,17 +56,26 @@
 
 EXAMPLES = '''
 # create a file
-- win_file: path=C:\\temp\\foo.conf 
+- win_file:
+    path: C:\temp\foo.conf
 
 # touch a file (creates if not present, updates modification time if present)
-- win_file: path=C:\\temp\\foo.conf state=touch 
+- win_file:
+    path: C:\temp\foo.conf
```

Comment:
I think a note here (and in all the windows modules) would be good.

We could maybe setup a docs_fragment note explaining this that we include in all of `windows/` modules. What do you think @nitzmahone  ?